home11.1/BookRecommender.java

```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class BookRecommender {
```

Comment:
the same concern as for class part: you can rearrange the code logic and avoid explicit exit calls in the botom levels of the program